extension/chrome/settings/setup.htm

```diff
@@ -229,12 +223,34 @@ <h1></h1>
       </div>
     </div>
 
+    <div id="step_2_ekm_choose_pass_phrase" class="manual">
+      <div class="line left">Choose pass phrase to protect your encrypted emails. <a href="#"
+          data-swal-page="/chrome/texts/passphrase_help.htm">choosing
+          secure pass phrases</a></div>
+      <div class="line left">
+        <input class="input_password" type="text" placeholder="Pass phrase" style="margin-bottom: 0;" id="step_2_ekm_input_password" data-test="input-step2ekm-passphrase-1" />
+      </div>
+      <div class="line left">
+        <input class="input_password2" type="text" placeholder="Repeat pass phrase" data-test="input-step2ekm-passphrase-2" id="step_2_ekm_input_password2" />
+      </div>
+
+      <div class="line left">
+        <label>
+          <input type="checkbox" class="input_passphrase_save" value="" name="check" data-test="input-step2ekm-save-passphrase" />
+          Remember Pass Phrase after closing browser
+        </label>
+      </div>
```

Comment:
Let's comment this out per another comment - in case we do want to bring it back in the future.